### ISSUE TITLE
yadage: write logs to a canonical file

### ISSUE THREAD

**Issue by diegodelemos** [reanahub/reana-workflow-engine-yadage#27]
Since a different endpoint for logs has been added (see https://github.com/reanahub/reana-job-controller/pull/41 and https://github.com/reanahub/reana-workflow-engine-yadage/pull/26) it would be nice to write [this content](https://github.com/reanahub/reana-workflow-engine-yadage/pull/26/files#diff-c64c72f14ae93d7fcae320831081e275R79) to a default file *à la* `HTCondor`:
```console
./log:
hello.47278.log

./output:
hello.47278.0.out
```

**Comment by lukasheinrich** [comment]
this will be the job of `reana-job-controller` but the engines can play the role of managing the locations to a canonical one that makes sense within a workflow

i.e.

```
engine --- submit job spec ---> job-controller
engine <--- retrieve logs ------ job-controller
the engine then writes the logs where it wants to
```

**Comment by diegodelemos** [comment]
yes, this is what you proposed last time, but it was not clear how to implement it with `yadage` back then, is it clear now?
